App.js

```diff
@@ -0,0 +1,93 @@
+const data = [
+  {image:"images/Portfoli33.png", title:"Tonic", info :['FACEBOOK',' Backend Dev','2015'], descript:"A daily selection of privately personalized reads; no accounts or sign-ups required.",
+  technologies :['Html',' css','Javascript']
+},
+
+{image:"images/Portfoli33.png", title:"Multi-Post Stories", info :['FACEBOOK',' Full stack Dev','2015'], descript:"A daily selection of privately personalized reads; no accounts or sign-ups required.",
+technologies :['Html',' css','Javascript']
+},
+
+{image:"images/Portfoli33.png", title:"Facebook 360", info :['FACEBOOK',' Full stack Dev','2015'], descript:"A daily selection of privately personalized reads; no accounts or sign-ups required.",
+technologies :['Html',' css','Javascript']
+},
+
+{image:"images/Portfoli33.png", title:"Uber Navigation", info :['FACEBOOK',' Full stack Dev','2015'], descript:"A daily selection of privately personalized reads; no accounts or sign-ups required.",
+technologies :['Html',' css','Javascript']
+}
+
+]
+//Create two varaible that holding hamburger button and Menu
+
+const hamburger = document.querySelector('.hamburger-btn');
+const navMenu = document.querySelector('.nav-menu');
+
+
+//Make event listeners
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  
+});
+
+
+//looping for nav link
+
+document.querySelectorAll('.anchor').forEach((navLink) => navLink.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    navMenu.classList.remove('active');
+   
+   
+  }));
+
+
+
+
+  //form validation in javascript
+ 
+  const form = document.querySelector('.form');
+
+  form.addEventListener('submit', (element) =>{
+    const email = document.querySelector('.email').value;
+    element.preventDefault();
+    if(email.toLowerCase() === email){
+      form.submit();
+    } else {
+      document.querySelector('.alert').textContent='please use lowercase !';
+    }
+  })
+
+
+  //Local storage
+
+  //
+
+  form.addEventListener('submit', (element) =>{
+
+    element.preventDefault();
+  
+      const nameField = document.querySelector('#name').value;
+      const emailField = document.querySelector('#email').value;
+      const textAreaField= document.querySelector("#textArea").value;
+  
+      localStorage.setItem('nameField',nameField);
+      localStorage.setItem('emailField',emailField);
+      localStorage.setItem('textAreaField',textAreaField);
+  
+      
+    
+    });
```

Comment:
- [ ] Please ensure that the data is stored in local storage when the user changes the content of any input fields not in the form submission as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=When%20the%20user%20changes%20the%20content%20of%20any%20input%20field%2C%20the%20data%20is%20saved%20to%20the%20local%20storage.).
- [ ] Please also make sure to store data in local storage as a single object as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=You%20must%20create%20a%20single%20JavaScript%20object%20with%20all%20the%20data%20from%20the%20entire%20form%20and%20save%20it%20in%20local%20storage.%20Do%20not%20create%20one%20local%20storage%20entry%20per%20input%20field.).